eras/allegra/impl/CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 
 ### `testlib`
 
+* Use named CDDL `slot` rule for timelock validity intervals
```

Comment:
Changelog needs to record only the visible changes relevant to the user. There is no need to record changes to implementation that are not visible outside or do not have any semantic differences.
That being said, with my other suggestion this change will have semantic meaning for CDDL
```suggestion
* Use fixed size `uint .size 8` for `slot` in CDDL for timelock validity intervals
```

Please, apply this suggestion to other similar changelog entries